src/components/card/BasicCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <el-card class="box-card">
```

Comment:
This can be removed now. As we are removing box card styling from style section

Reply:
Good catch! Thanks